src/Strategy/Python/Util.hs

```diff
@@ -269,9 +269,12 @@ requirementParser = specification
 
     marker_or :: Parser Marker
     marker_or =
-      label "marker_or" $
-        try (MarkerOr <$> marker_and <* whitespace <* string "or" <*> marker_and)
-          <|> marker_and
+      label "marker_or" $ do
```

Comment:
I think this still fails for mulitple `and`s in a line. E.g. this fails:

```
    "greenlet!=0.4.17; (platform_machine == \"x\" or platform_machine == \"a\" or platform_machine == \"b\") and platform_release < \"3.14\" and platform_version < \"3.1.4\"",
```

If I change either of the `and`s to an `or`, it parses:

```
    "greenlet!=0.4.17; (platform_machine == \"x\" or platform_machine == \"a\" or platform_machine == \"b\") or platform_release < \"3.14\" and platform_version < \"3.1.4\"",
```

I'm not sure that this is blocking, though. This currently fixes the customer's problem, so we could ship it. But if it's easy to fix the problem then I'd do that